src/meta-data/message-broker.py

```diff
@@ -0,0 +1,189 @@
+                model.get_distance_to_closest_residential_road(
+                    site_latitude, site_longitude
+                )
+            )
+            print("Computing distance to closest tertiary road.....")
+            distance_to_closest_tertiary_road = (
+                model.get_distance_to_closest_tertiary_road(
+                    site_latitude, site_longitude
+                )
+            )
+            print("Computing distance to closest trunk.....")
+            distance_to_closest_trunk = model.get_distance_to_closest_trunk(
+                site_latitude, site_longitude
+            )
+            print("Computing distance to closest unclassified road.....")
+            distance_to_closest_unclassified_road = (
+                model.get_distance_to_closest_unclassified_road(
+                    site_latitude, site_longitude
+                )
+            )
+            print("Computing distance to closest motorway.....")
+            distance_to_closest_motorway = model.get_distance_to_closest_motorway(
+                site_latitude, site_longitude
+            )
+            print("Saving distances.....")
+            airqo_api.update_site_meta_data(
+                {
+                    **site_meta_data,
+                    **{
+                        "distance_to_kampala_center": distance_from_kampala,
+                        "distance_to_nearest_road": distance_to_closest_road,
+                        "distance_to_nearest_secondary_road": distance_to_closest_secondary_road,
+                        "distance_to_nearest_primary_road": distance_to_closest_primary_road,
+                        "distance_to_nearest_residential_road": distance_to_closest_residential_road,
+                        "distance_to_nearest_tertiary_road": distance_to_closest_tertiary_road,
+                        "distance_to_nearest_trunk": distance_to_closest_trunk,
+                        "distance_to_nearest_unclassified_road": distance_to_closest_unclassified_road,
+                        "distance_to_nearest_motorway": distance_to_closest_motorway,
+                    },
+                }
+            )
+
+            print("Computing land use.....")
+            land_use = model.get_landuse(site_latitude, site_longitude)
+            print("Saving land use.....")
+            airqo_api.update_site_meta_data(
+                {
+                    **site_meta_data,
+                    **{
+                        "land_use": land_use,
+                    },
+                }
+            )
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--target",
+        required=True,
+        type=str.lower,
+        choices=[
+            "air-qlouds-consumer",
+            "sites-consumer",
+            "devices-consumer",
+        ],
+    )
+
+    args = parser.parse_args()
+    if args.target == "air-qlouds-consumer":
```

Comment:
This is good. I was about to talk about a need for us to also consider scalable approaches which support the consumption of different Kafka topics. Thanks @NoahNsimbe